app/src/main/java/com/celzero/bravedns/ui/HomeScreenFragment.kt

```diff
@@ -486,8 +541,8 @@ class HomeScreenFragment : Fragment(R.layout.fragment_home_screen) {
             val prepareVpnIntent: Intent? = try {
                 VpnService.prepare(context)
             } catch (e: NullPointerException) {
-               Log.w(LOG_TAG,"Prepare VPN Exception - ${e.message}")
-               null
+                Log.w(LOG_TAG, "Prepare VPN Exception - ${e.message}")
```

Comment:
Log the exception `Log.w(tag, msg, ex)`

when / why do we hit NPE?

Reply:
Added the logs. 
For NPE - Something similar to this. - [1084](https://github.com/AdguardTeam/AdguardForAndroid/issues/1084)

Reply:
Then, add that ref or some other ref like that in the code comment?

Reply:
Added as part of code comment.